vm/os-windows.cpp

```diff
@@ -296,6 +296,50 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+const int ctrl_break_sleep = 10; /* msec */
```

Comment:
Doesn't seem like we should check for interrupted so frequently.  When you press ``Ctrl-Break``, it doesn't have to be "immediate", it can be "almost immediate".

Also, wasn't a no-polling patch in the works?  I recall you mentioning that at some point?